src/Tribe/REST/V1/Endpoints/Single_Attendee.php

```diff
@@ -81,8 +81,15 @@ public function READ_args() {
 
 	/**
 	 * {@inheritdoc}
+	 *
+	 * @since TBD Returns 401 Unauthorized if Event Tickets Plus is not loaded.
 	 */
 	public function get( WP_REST_Request $request ) {
+		// Early bail: ET Plus must be active to use this endpoint.
+		if ( ! class_exists( "Tribe__Tickets_Plus__Main" ) ) {
```

Comment:
:no_entry_sign: String "Tribe__Tickets_Plus__Main" does not require double quotes; use single quotes instead

<sub>hash: e35cbe35ffa3392d962fdd23f9c000c2</sub>